docs/install/install-troubleshooting.md

```diff
@@ -90,11 +95,27 @@ All containers: docker-compose logs --tail=0 --follow
 
 ---
 
-Pick one of the following:
-**Return to [Demo ISLE Installation](../install/install-demo.md).**
-**Return to [Local ISLE Installation: New site](../install/install-local-new.md).**
-**Return to [Local ISLE Installation: Migrate existing Islandora site](../install/install-local-migrate.md).**
-**Return to [Staging ISLE Installation: New site](../install/install-staging-new.md).**
-**Return to [Staging ISLE Installation: Migrate existing Islandora site](../install/install-staging-migrate.md).**
-**Return to [Production ISLE Installation: New site](../install/install-production-new.md).**
-**Return to [Production ISLE Installation: Migrate existing Islandora site](../install/install-production-migrate.md).**
\ No newline at end of file
+## Fedora Hash Size (Conditional)
+
+**Are you migrating an existing Islandora site that has greater than one million objects?**
+
+This is a power user setting and is an incredibly rare sitation, so do this step only if you have an akubra level 2 or greater.
+
+While this will depend on your pre-existing Production system, it is important to double-check this. If you have a larger hash size than the default ISLE system (##), and don't follow the settings below, ISLE may not function properly when your data has been migrated.
```

Comment:
Should the (##) be filled in with a value, presumably for the default hash size?

Reply:
I've noted this in my list of future doc edits. I'll follow up on correcting this item in a future PR.